tests/conftest.py

```diff
@@ -0,0 +1,69 @@
+from virtwho.settings import config
+from virtwho.runner import VirtwhoRunner
+from virtwho.configure import VirtwhoHypervisorConfig
+from virtwho.configure import VirtwhoGlobalConfig
+from virtwho.configure import get_hypervisor_handler
+from virtwho.configure import get_register_handler
+from virtwho.register import SubscriptionManager
+import pytest
+
+
+mode = config.job.mode
+register_type = config.job.register
```

Comment:
not make sense to me, we have a new section `job`?

Reply:
Sorry, I added it locally and then forgot to push and discuss it with you. My though is to fill in the ```mode``` and ```register``` to virtwho.ini, then direct to read and navigate to the section.

```
[job]
mode=
register=
```